resourcemanager/schedule.go

```diff
@@ -25,8 +25,11 @@ import (
 
 func (r *ResourceManager) schedule() {
 	r.poolMap.Iter(func(pool *util.PoolContainer) {
+		if pool.Component == util.DistTask {
+			return
+		}
```

Comment:
Skip disttask  pool to schedule.

Reply:
The business side does not need to schedule. It is to face the local resource. not disturb resources.